src/pages/About.jsx

```diff
@@ -48,6 +48,17 @@ const team = [
       </>
     ),
   },
+  {
+    name: 'Job Hernandez',
+    img: testPic,
+    description: (
+      <>
+        I am interested and driven to solve problems with software; software has
+        tremendous power to make an impact in the world and transform lives.
+      </>
+    ),
+  },
+
```

Comment:
Gotta update your branch to make these changes disappear. GitHub has a button for this on the main PR tab but otherwise you can do `git pull origin main` to merge changes from the `main` branch on GitHub.